liqd_product/apps/partners/templates/partner_landing_page.html

```diff
@@ -0,0 +1,20 @@
+{% extends 'partner_base.html' %}
+{% load thumbnail %}
+
+{% block title %}{{partner.name}}{% endblock %}
+
+{% block content %}
+    <div class="hero-unit" style="background-image: url({% thumbnail partner.image "1500x610" crop %});">
+        <h1 class="hero-unit__title">
+            {{ partner.slogan }}
+        </h1>
```

Comment:
This should not be `h1` because it is not a heading for the following content. I would just use a `div` instead.